pkg/client/core/store/filesystem/externalsecrets.go

```diff
@@ -30,12 +30,15 @@ func (s *externalSecretsStore) RemoveExternalSecrets(_ api.ID) (*store.Report, e
 		Remove(s.chart.OutputFile).
 		Remove(s.chart.ReleaseFile).
 		Remove(s.chart.ChartFile).
-		Remove("").
 		Do()
 	if err != nil {
 		return nil, err
 	}
 
+	_, _ = store.NewFileSystem(s.chart.BaseDir, s.fs).
```

Comment:
In this and the other similar commands, are we not ignoring errors, which may cause to panic?

Reply:
We ignore errors, so it will just continue if there is an error

Reply:
What happened last time was that delete execution stopped since a directory was not empty - and we had told it to return that error - now we just ignore it. 
The linter won't let you just
"store.NewFileSystem(s.chart.BaseDir, s.fs).Remove("").Do()"

Reply:
But we can't go around and just ignore errors, can we? :smile: If there's an error creating the filesystem, we can't just use as if nothing happened?

Reply:
I agree, this ought to be rewritten, we have taken on some technical debt to get delete cluster functional :)